stacker/providers/aws/interactive.py

```diff
@@ -80,7 +80,7 @@ def output_summary(fqn, action, changeset, replacements_only=False):
     changes = []
     for change in changeset:
         resource = change['ResourceChange']
-        replacement = resource['Replacement'] == 'True'
+        replacement = resource.get('Replacement', '') == 'True'
```

Comment:
Yeah - I think I prefer just letting it use the usual default, unless there's actually something useful about specifying a new default.  If you can update that, I'll pull this in ASAP :)  Thanks @danielkza